cli/main.rs

```diff
@@ -1140,6 +1157,11 @@ async fn run_command(
   flags: Flags,
   run_flags: RunFlags,
 ) -> Result<i32, AnyError> {
+  if !flags.has_check_flag && flags.type_check_mode == TypeCheckMode::All {
+    info!("{} In future releases deno run will not automatically type check without the --check flag. 
```

Comment:
```suggestion
    info!("{} In future releases `deno run` will not automatically type check without the --check flag. 
```

Does this respect `--quiet`?

Reply:
Yes, it does